xform-marc21-to-xml/src/main/java/edu/stanford/AuthDBProperties.java

```diff
@@ -76,6 +80,37 @@ private void initDataSourceProperties(Properties properties) {
         this.userPass = properties.getProperty("PASS");
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Consider related commentary, e.g.
- http://strugglingwithruby.blogspot.com/2010/04/operator-overloading.html
- http://stackoverflow.com/questions/11186815/override-operator-in-ruby

This might have to change from overloading `equals` to to overload `==` instead.